domains/nerd.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "xkcdnerd",
+           "email": "evanwang330@gmail.com",
+           "discord": "1153506811522076723"
+        },
+    
+        "record": {
+            "CNAME": "nerd.is-a.dev"
```

Comment:
you cant have the CNAME record as the domain you want to get

Reply:
fixed it